### DISCUSSION TITLE
PostgreSQL issue

### DISCUSSION THREAD

**Discussion by ashwintemkar** [keyshade-xyz/keyshade#508] [General]
Hi everyone,

I’ve been working on setting up the project according to the [guidelines](https://github.com/keyshade-xyz/keyshade/blob/develop/CONTRIBUTING.md), but I’ve encountered a recurring problem that I believe many contributors might also be facing.

below is the screenshot attached for reference

![Screenshot 2024-10-22 191849](https://github.com/user-attachments/assets/74abeeb7-c237-46d4-86af-8dc9144bbf04)

Despite following the recommendations regarding [PostgreSQL issues](https://docs.keyshade.xyz/contributing-to-keyshade/setting-things-up#issues-regarding-postgresql), I did not find any conculsion to this

Hence, I think it would be beneficial for all of us to address this issue collectively, as it seems to affect multiple contributors.

Looking forward to any insights or solutions you might have!

thanks to @rajdip-b who called up to open a [discussion ](https://github.com/keyshade-xyz/keyshade/issues/406#issuecomment-2432435480)on this 

Please upvote if you also faced / are facing the same

**Comment by rajdip-b** [comment]
Thanks for opening up this discussion! 

You mentioned that you are using WSL. Any special reason as to why you are using that? Also, could you give the screenshot of this command:
```bash
docker ps
```

**Reply by ashwintemkar** [reply]
sure @rajdip-b 
below is the ss for `docker ps`
![image](https://github.com/user-attachments/assets/51668422-4d38-402d-b656-102d3655a3a6)

**Reply by ashwintemkar** [reply]
and the reason to use WSL is it requires fewer resources than a full virtual machine, and I have a low end pc, also I don't use powershell, hence prefer shell.

**Reply by rajdip-b** [reply]
![image](https://github.com/user-attachments/assets/64d5885a-c4d7-4aef-80ae-54a3538f991a)

As you can see, you would need to spin down Strapi resources. You can either do `docker compose down` in your fork folder for Strapi, or you can do `docker stop 50 1b`. After that, you can use `docker compose up -d` in keyshade.

**Reply by ashwintemkar** [reply]
Hey @rajdip-b thank you so much for noticing that, I was narrow minded at that observation.

thanks, now I was able to deploy the migrations

**Reply by rajdip-b** [reply]
Haha glad to know that it worked!
